source/docs/advanced/payments/adyen.md

```diff
@@ -5,46 +5,67 @@ title: Adyen integration
-
-### Install and configure the `Adyen_Payment` Magento2 extension (7.2+)
-
-Follow [Adyen's documentation](https://docs.adyen.com/plugins/magento-2) to install and configure the official Magento2 extension for Adyen payments. The minimum supported version is [7.2.0](https://github.com/Adyen/adyen-magento2/releases/tag/7.2.0).
+### Add the required environment variables
 
-You must configure the "Payment Origin URL" in the "Advanced: PWA" with your Front-Commerce URL **for each of your stores**.
+You need to define the following environment variables:
 
-<blockquote class="important">
-**IMPORTANT** please note that enabling or disabling payments in Magento's admin area has no effect on payment methods visible on the storefront. This is a *feature* from the module: every active payment methods in Adyen will be available. **You either have to filter them in the frontend or from your Adyen account.**
-</blockquote>
-
-### Add the Adyen client key in the environment
-
-You need to define the `FRONT_COMMERCE_ADYEN_CLIENT_KEY` environment variable so that it contains [your Adyen client key](https://docs.adyen.com/development-resources/client-side-authentication) for the domains of your Front-Commerce stores:
+* `FRONT_COMMERCE_ADYEN_MERCHANT_ACCOUNT` your merchant account name. You can find it in your adyen Customer Area in the top left corner next to your company name.
+* `FRONT_COMMERCE_ADYEN_CLIENT_KEY` environment variable so that it contains [your Adyen client key](https://docs.adyen.com/development-resources/client-side-authentication) for the domains of your Front-Commerce stores. You can find it in your adyen Customer Area under `Developers > API Credential > [your_prefered_credential] > Client Key > Client Key`.
+* `FRONT_COMMERCE_ADYEN_API_KEY` you api key. You can find it in your adyen Customer Area under `Developers > API Credential > [your_prefered_credential] > API Key > API Key`.
+* `FRONT_COMMERCE_ADYEN_LIVE_URL_PREFIX` only needed in production environment. Should be configured to contain [the adyen live URL prefix](https://docs.adyen.com/development-resources/live-endpoints#live-url-prefix).
+* `FRONT_COMMERCE_ADYEN_NOTIFICATION_USERNAME` (more on webhooks below) you create it. You then have to copy it to the webhook section in your adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Authentication > User Name`.
+* `FRONT_COMMERCE_ADYEN_NOTIFICATION_PASSWORD` (more on webhooks below) you create it. You then have to copy it to the webhook section in your adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Authentication > Password`.
+* `FRONT_COMMERCE_ADYEN_HMAC_KEY` (more on webhooks below) create it from the webhook section in your adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Additional Settings > HMAC key`.
+* `FRONT_COMMERCE_ADYEN_PREVIOUS_HMAC_KEY` (more on webhooks below) when you regenerate your HMAC key configure this to be the old one.
```

Comment:
Just ensure that the hash is correct, but it should slugify to that based on the title

```suggestion
* `FRONT_COMMERCE_ADYEN_NOTIFICATION_USERNAME` (more on [webhooks](#add-webhook) below) you create it. You then have to copy it to the webhook section in your adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Authentication > User Name`.
* `FRONT_COMMERCE_ADYEN_NOTIFICATION_PASSWORD` (more on [webhooks](#add-webhook) below) you create it. You then have to copy it to the webhook section in your adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Authentication > Password`.
* `FRONT_COMMERCE_ADYEN_HMAC_KEY` (more on [webhooks](#add-webhook) below) create it from the webhook section in your adyen Customer Area under `Developers > Webhooks > [your_prefered_webhook] > Additional Settings > HMAC key`.
* `FRONT_COMMERCE_ADYEN_PREVIOUS_HMAC_KEY` (more on [webhooks](#add-webhook) below) when you regenerate your HMAC key configure this to be the old one.
```